workbench/src/renderer/components/OpenButton/index.jsx

```diff
@@ -38,6 +38,8 @@ export default class OpenButton extends React.Component {
     const tipText = _('Browse to a datastack (.json) or InVEST logfile (.txt)');
     return (
       <OverlayTrigger
+        // defaultShow={this.props.defaultShow}
+        // flip={this.props.defaultShow}
```

Comment:
Can these commented lines be deleted?

Reply:
Oh thanks! I was trying something out but ultimately made no changes to this.